CharlieBackend.Api.UnitTes/EventServiceTest.cs

```diff
@@ -0,0 +1,176 @@
+            _themeRepositoryMock = new Mock<IThemeRepository>();
+            _groupRepositoryMock = new Mock<IStudentGroupRepository>();
+            _validEvent = new ScheduledEvent
+            {
+                StudentGroupId = 1,
+                ThemeId = 1,
+                MentorId = 1
+            };
+            _existingId = 551;
+            _nonexistingId = 999;
+        }
+
+        [Fact]
+        public async Task GetEvent_ExistingId_ShouldReturnScheduledEvent() 
+        {
+            //Arrange
+            _eventRepositoryMock.Setup(x => x.GetByIdAsync(_existingId))
+                .ReturnsAsync(_validEvent);
+
+            _unitOfWorkMock.Setup(x => x.ScheduledEventRepository).Returns(_eventRepositoryMock.Object);
+
+            var eventService = new EventsService(_unitOfWorkMock.Object, _mapper);
+
+            //Act
+            var successResult = await eventService.GetAsync(_existingId);
+
+            //Assert
+            successResult.Should().NotBeNull();
+            successResult.Should().BeEquivalentTo(_mapper.Map<ScheduledEventDTO>(_validEvent));
+        }
+
+        [Fact]
+        public async Task GetEvent_NonExistingId_ShouldReturnNull()
+        {
+            //Arrange
+            _eventRepositoryMock.Setup(x => x.GetByIdAsync(_existingId))
+                .ReturnsAsync(_validEvent);
+
+            _unitOfWorkMock.Setup(x => x.ScheduledEventRepository).Returns(_eventRepositoryMock.Object);
+
+            var eventService = new EventsService(_unitOfWorkMock.Object, _mapper);
+
+            //Act
+            var successResult = await eventService.GetAsync(_nonexistingId);
+
+            //Assert
+            successResult.Should().BeNull();
+        }
+
+        [Fact]
+        public async Task UpdateEvent_ValidData_ShouldReturnScheduledEvent()
+        {
+            //Arrange
+            _eventRepositoryMock.Setup(x => x.GetByIdAsync(_existingId))
+                .ReturnsAsync(_validEvent);
+            _mentorRepositoryMock.Setup(x => x.GetByIdAsync(1))
+                .ReturnsAsync(new Mentor { AccountId = 111 });
+            _themeRepositoryMock.Setup(x => x.GetByIdAsync(222))
+                .ReturnsAsync(new Theme { Id = 1 });
+            _groupRepositoryMock.Setup(x => x.GetByIdAsync(1))
+                .ReturnsAsync(new StudentGroup { Id = 1 });
+
+            _unitOfWorkMock.Setup(x => x.ScheduledEventRepository).Returns(_eventRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.MentorRepository).Returns(_mentorRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.ThemeRepository).Returns(_themeRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.StudentGroupRepository).Returns(_groupRepositoryMock.Object);
+
+            var eventService = new EventsService(_unitOfWorkMock.Object, _mapper);
+            
+            _update = new UpdateScheduledEventDto
+            {
+                StudentGroupId = 1,
```

Comment:
Please declare variables for these numbers (studentGroupId, themeId, mentorId) in the Arrange section.